app/templates/upfronts_table.html

```diff
@@ -0,0 +1 @@
+works
```

Comment:
Do you want to push this? Treat every PR as a production-ready code. 
IMO you should push at least a template page like emmet auto-generated code. See this cheat-sheet and check if your IDE can make this for you.
https://docs.emmet.io/cheat-sheet/